google/cloud/bigquery/v2/minimal/internal/job_options.cc

```diff
@@ -26,8 +27,24 @@ GOOGLE_CLOUD_CPP_INLINE_NAMESPACE_BEGIN
 
 namespace {
 auto constexpr kBackoffScaling = 2.0;
+std::size_t constexpr kJobDefaultConnectionPoolSize = 4;
+std::size_t constexpr kJobDefaultConnectionPoolSizeMax = 64;
 }  // namespace
 
+std::size_t DefaultConnectionPoolSize() {
```

Comment:
It is a little weird to have a `kFoo` and then a function `Foo()` that doesn't just return `kFoo`.

That is, if the function is `DefaultConnectionPoolSize()`, then perhaps its default value should be `kDefaultDefaultConnectionPoolSize`.

And from there you might choose to remove one `Default` from each name.